docs/examples/api/v1/users/contacts/fetch_contact/fetch_contact.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	wallet "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet-go-client/docs/examples"
+	"github.com/bitcoin-sv/spv-wallet-go-client/docs/examples/exampleutil"
+)
+
+func main() {
+	spv, err := wallet.NewWithXPriv(exampleutil.ExampleConfig, examples.XPriv)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	paymail := "john.doe@example.com"
+	contact, err := spv.ContactWithPaymail(context.Background(), paymail)
```

Comment:
I haven't noticed this before but this might be better called `ContactByPaymail`? Reading this eample I thought we will receive a contact with paymail field and it doesn't match exactly what the method do. WDYT?

Reply:
Thanks @dzolt-4chain , good point. I'll fix it in the separate task after merging this and Admin Access API PRs.